bin/apisix

```diff
@@ -50,6 +50,12 @@ if script_path:sub(1, 2) == './' then
         error("failed to fetch current path")
     end
 
+    is_root_path = string.match(apisix_home, "^\/[root][^\/]+")
```

Comment:
```
./bin/apisix init
lua: ./bin/apisix:53: invalid escape sequence near '"^\/'
make: *** [Makefile:77: init] Error 1
```

Reply:
aw, which OS are you using?

Reply:
Can I know which Lua version are you using, which prompts this issue?

Reply:
#  lua -v
Lua 5.3.5  Copyright (C) 1994-2018 Lua.org, PUC-Rio

Reply:
Thanks, 5.1 seems to ignore this error.

Reply:
Fixed for Lua 5.3